HRCounter/HRCounter.csproj

```diff
@@ -55,9 +55,9 @@
       <HintPath>$(BeatSaberDir)\Plugins\Counters+.dll</HintPath>
       <SpecificVersion>False</SpecificVersion>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+    <Reference Include="Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
       <Private>False</Private>
-      <HintPath>$(BeatSaberDir)\Libs\Newtonsoft.Json.dll</HintPath>
+      <HintPath>$(BeatSaberDir)\Beat Saber_Data\Managed\Newtonsoft.Json.dll</HintPath>
```

Comment:
These two changes are not necessary anymore because the new BSIPA will force the 12.0 version to load first i.e the one in the `Libs` folder

Reply:
Reverted Newtonsoft.Json changes in csproj